### ISSUE TITLE
OS suspension

### ISSUE THREAD

**Issue by Andrey1386** [anegostudios/VintageStory-Issues#2383]
### Game Version

net7-v.1.17.11-rc.2

### Platform

Linux

### Modded

Modded

### SP/MP

Singleplayer

### Description

When you start the program, after ~5 seconds the login window appears, the os completely freezes. 

I use:
HW: Acer Nitro 5 AN515-57-548J, i5-11400H 64 Gb, RTX 3050 TI 8Gb
OS: Kubuntu 22.04

dotnet --list-runtimes
Microsoft.AspNetCore.App 7.0.3 [/usr/lib/dotnet/shared/Microsoft.AspNetCore.App]
Microsoft.NETCore.App 7.0.3 [/usr/lib/dotnet/shared/Microsoft.NETCore.App]

I have this problem with some other wine-launched games (not all).
I have never seen such a problem with Vintagestory before.

The laptop, meanwhile, works, plays music, plays a video (audio track) in the background, responds to ssh commands, when you try to kill the application, the freeze does not disappear. dmesg not reporting errors.

PS: I don't speak English well, so I apologize if something is not written clearly.

### How to reproduce

1. command: DRI_PRIME=1 __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia ./Vintagestory
2. Wait around 5 seconds
3. Video freezes, only cursor movement remains, switching virtual desktops or opening an application does not work, does not respond to hot keys, even Ctrl + Alt + Fn

### Logs

```Log```

**Comment by Th3Dilli** [comment]
i am not sure how to reproduce this since i dont have a laptop with integrated gpu and dedicated gpu

so if you just run it without the special environment variables it will only use the integrated GPU i guess?
do you have nvidia-driver-515 isntalled?

**Comment by Andrey1386** [comment]
1. Yes. run it without the special environment variables it will only use the integrated GPU.
2. nvidia-driver-515 installed.
3. I think the problem is with the graphics card driver. Sometimes, after updating the driver, some games start working without problems. I just wrote that unix users will have problems running the game on laptops leading to a freeze. I described the freezing of the linux kernel.
